### ISSUE TITLE
Align wrapped lines with indented text in lists

### ISSUE THREAD

**Issue by aaaron** [yzhang-gh/vscode-markdown#824]
#### What is the problem?

(Thanks for your hard work on this fantastic extension!)

I frequently use lists to organize my notes. I'm a transplant from Emac's Org Mode, where long lines are neatly wrapped to line up with text (after the bullet). In Markdown edit, the long lines wrap aligned with the bullet, which is visually hard to parse. (Of course, this is only a problem in edit mode, not preview.) Wondering if this is something that can be addressed through a feature request, existing setting, or is just fundamentally counter to how VSCode operates...

#### How can I reproduce it?

[
<img width="439" alt="indented_wrap" src="https://user-images.githubusercontent.com/565405/94063095-9b062300-fdb5-11ea-8a29-387469037e42.png">
](url)

**Comment by yzhang-gh** [comment]
Thanks! It's very generous of you.

This is something controlled by VSCode itself. You can play around with the `editor.wrappingIndent` option.

**Comment by aaaron** [comment]
`"editor.wrappingIndent": "indent",`

Perfect, thanks!
